CondFormats/GeometryObjects/interface/HcalParameters.h

```diff
@@ -64,6 +64,7 @@ class HcalParameters {
   std::vector<int>    phigroup;
   std::vector<LayerItem> layerGroupEtaSim, layerGroupEtaRec;
   int                 topologyMode;
+  int                 triggerMode;
```

Comment:
Thanks - I shall do that. I shall give them 8 bits each right now.
________________________________
From: Ianna Osborne [notifications@github.com]
Sent: 09 November 2015 16:35
To: cms-sw/cmssw
Cc: Sunanda Banerjee
Subject: Re: [cmssw] bsunanda:Run2-hcx47 Add trigger primitive version in geometry (#12301)


In CondFormats/GeometryObjects/interface/HcalParameters.h<https://github.com/cms-sw/cmssw/pull/12301#discussion_r44288078>:

> @@ -64,6 +64,7 @@ class HcalParameters {
>    std::vector<int>    phigroup;
>    std::vector<LayerItem> layerGroupEtaSim, layerGroupEtaRec;
>    int                 topologyMode;
> +  int                 triggerMode;


@bsunanda<https://github.com/bsunanda> - this will require schema change. I'd suggest to use the topologyMode integer with mask to store/retrieve triggerMode.

—
Reply to this email directly or view it on GitHub<https://github.com/cms-sw/cmssw/pull/12301/files#r44288078>.